src/AbstractTooltip/index.tsx

```diff
@@ -52,8 +52,14 @@ export const AbstractTooltip: React.FC<Props> = ({
         className={classnames(className, {
           "space-kit-relaxed": padding === "relaxed",
         })}
-        duration={disableAnimations ? 0 : props.duration}
-        updateDuration={disableAnimations ? 0 : props.updateDuration}
+        duration={
+          (disableAnimations ? 0 : props.duration) ??
```

Comment:
I think the () might be more clear wrapping `(props.duration ?? tippy.defaultProps.duration)` since thats what were actually falling through

Reply:
Good idea! I'll do that

Reply:
When I did that eslint got rid of it because then I'm no longer overriding default behavior 🤷‍♂

Reply:
That's still better though, even if we have to look up order of operations with `??`.